src/utils/blockchain/contracts.testnet.ts

```diff
@@ -324,7 +324,7 @@ export const contracts = {
     chainId: ChainId.BSC_TESTNET,
   },
   PERPETUALS_token: {
-    address: '0xd0370a808203da14B703826eF77072ef5F09840D',
+    address: '0xcF3D22A034Fa157985F0Fe71F15477446f80Be26',
```

Comment:
I wouldn't adjust it in this PR because even though we can easily test BTC Deposit/Withdraw, we won't be able to trade until we have a new perps deployment that uses the new contract address.

Reply:
Understood, maybe you have expected date when perps will have new contracts? If it's soon I think we can leave this change here, and swap it back only if PR will be merged before contracts are upgraded.

Reply:
Once tudor solves bridging problem we have, it shouldn't take too long. So I think it's ok to leave it there,